iam/api-client/src/test/java/com/google/iam/snippets/GrantableRolesIT.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.iam.snippets;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class GrantableRolesIT {
+
+  @Test
+  public void testQuickstart() throws Exception {
+    String project = System.getenv("GOOGLE_CLOUD_PROJECT");
+    String resource = "//cloudresourcemanager.googleapis.com/projects/" + project;
+
+    GrantableRoles.main(new String[] {resource});
```

Comment:
Can you intercept the output of the program running and then verify that the output is what you'd expect? There are several examples of doing this in other IT's in this repo, just search for "bout" and you'll find one.

Reply:
Done